lib/libutils/isoc/sprintf.c

```diff
@@ -17,3 +18,16 @@ int sprintf(char *str, const char *fmt, ...)
 
 	return retval;
 }
+
+int __sprintf_chk(char *str, int flag __unused, size_t slen __unused,
+		  const char *fmt, ...)
+{
+	int retval;
+	va_list ap;
+
+	va_start(ap, fmt);
+	retval = __vsprintf(str, fmt, ap);
```

Comment:
Surely `vsnprintf()` is a better candidate to call here. Then we'd be able to check overflow in `str` which i guess is the most common error.

Reply:
Makes sense, I will switch to `vsnprintf()`.